core/templates/AppSpec.ts

```diff
@@ -28,9 +28,10 @@ import { WrittenTranslationsObjectFactory } from
 import { UpgradedServices } from 'services/UpgradedServices';
 // ^^^ This block is to be removed.
 
-import constants from 'assets/constants';
+import { AppConstants } from 'app.constants';
 import sourceMappedStackTrace from 'sourcemapped-stacktrace';
 
+const constants = AppConstants.commonConstants;
```

Comment:
Ah, can you remove this and replace all occurrences of `constants` in this file with `AppConstants`.

Reply:
ditto in other files

Reply:
Hi, I solve some conflicts with new change and this PR is closed automatically, I will rebase new changes and open a new PR.

Reply:
https://github.com/oppia/oppia/pull/17084

Reply:
I don't think it was closed automatically, it seems that you closed it yourself.

![image](https://user-images.githubusercontent.com/10142938/213806839-5f2a49de-61e6-4872-a426-1c29cf56144f.png)

Reply:
> I don't think it was closed automatically, it seems that you closed it yourself.
> 
> ![image](https://user-images.githubusercontent.com/10142938/213806839-5f2a49de-61e6-4872-a426-1c29cf56144f.png)

After I rebase all commits it closed.

Reply:
Just a note, please never rebase. This typically involves modifying history which makes it hard for the reviewer, so we generally prohibit it.

In general make sure to follow the instructions here carefully: https://github.com/oppia/oppia/wiki/Make-a-pull-request

Thanks!

Reply:
> Just a note, please never rebase. This typically involves modifying history which makes it hard for the reviewer, so we generally prohibit it.
> 
> In general make sure to follow the instructions here carefully: https://github.com/oppia/oppia/wiki/Make-a-pull-request
> 
> Thanks!

Ok, I will make sure not do this in the furture.